Mathlib/MeasureTheory/OuterMeasure/OfFunction.lean

```diff
@@ -90,6 +90,25 @@ theorem ofFunction_apply (s : Set α) :
     OuterMeasure.ofFunction m m_empty s = ⨅ (t : ℕ → Set α) (_ : s ⊆ iUnion t), ∑' n, m (t n) :=
   rfl
 
+/-- Same as `ofFunction_apply`, except that the sets `t i` satisfy a predicate `P`,
```

Comment:
Can you also explain in the docstring what the theorem does, without referring to the other lemma? When they appear in tooltips, it's better if the docstrings are self-contained.